scripts/apex/hello.apex

```diff
@@ -1,10 +1 @@
-// Use .apex files to store anonymous Apex.
-// You can execute anonymous Apex in VS Code by selecting the
-//     apex text and running the command:
-//     SFDX: Execute Anonymous Apex with Currently Selected Text
-// You can also execute the entire file by running the command:
-//     SFDX: Execute Anonymous Apex with Editor Contents
-
-string tempvar = 'Enter_your_name_here';
-System.debug('Hello World!');
-System.debug('My name is ' + tempvar);
\ No newline at end of file
+FlowControlLoopsCollections.sumUpToLimit(5);
```

Comment:
Don't commit extraneous files to your branch. It's fine to have these files, just don't add them to the PR.